Apps/Common/EqualOrderOperators.h

```diff
@@ -183,7 +183,7 @@ class EqualOrderOperators
       for (size_t i = 1;i <= fe.basis(basis).size();i++) {
         for (size_t k = 1;k <= cmp;k++) {
           double diff = 0.0;
-          for (size_t l = 1;l <= cmp;l++)
+          for (size_t l = 1;l <= fe.grad(basis).cols();l++)
```

Comment:
Yes, now it makes (more) sense. And you're right, I read it as 'use cmp instead of nsd' before.
(I prefer for live in the present, not in the past ;-)